### ISSUE TITLE
Lucide version 0.15 not work in angular

### ISSUE THREAD

**Issue by SMAH1** [lucide-icons/lucide#273]
## Prerequisites

Version: 0.15.0
Running from: angular 11 (webpack)
Operating system: linux
Bits: 64 bits

## Description

After add `lucide` to project and use it like:
``` js
import { Component, OnInit } from '@angular/core';
import { Activity, createElement } from 'lucide';

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.css']
})
export class AppComponent implements OnInit {

  ngOnInit(): void {
    const ico = createElement(Activity);
  }

}
```

show error:

```
Error: node_modules/lucide/dist/lucide.d.ts:3:54 - error TS2304: Cannot find name 'SVGProps'.

3 export type IconNode = readonly [tag: string, object:SVGProps<SVGSVGElement>, children:IconNode?];
                                                       ~~~~~~~~


Error: node_modules/lucide/dist/lucide.d.ts:3:88 - error TS5086: A labeled tuple element is declared as optional with a question mark after the name and before the colon, rather than after the type.

3 export type IconNode = readonly [tag: string, object:SVGProps<SVGSVGElement>, children:IconNode?];
                                                                                         ~~~~~~~~~


Error: node_modules/lucide/dist/lucide.d.ts:4:26 - error TS1170: A computed property name in a type literal must refer to an expression whose type is a literal type or a 'unique symbol' type.

4 export type IconsObj = { [IconName]: IconNode }
                           ~~~~~~~~~~


Error: node_modules/lucide/dist/lucide.d.ts:4:27 - error TS2693: 'IconName' only refers to a type, but is being used as a value here.

4 export type IconsObj = { [IconName]: IconNode }
                            ~~~~~~~~


Error: node_modules/lucide/dist/lucide.d.ts:6:45 - error TS2304: Cannot find name 'Props'.

6 export interface Attributes extends Partial<Props<Element>> {}
                                              ~~~~~


Error: node_modules/lucide/dist/lucide.d.ts:9:58 - error TS2371: A parameter initializer is only allowed in a function or constructor implementation.

9 export function createIcons({ icons: IconsObj, nameAttr: string = 'icon-name', attrs: Attributes = {} }): VoidFunction;
                                                           ~~~~~~


Error: node_modules/lucide/dist/lucide.d.ts:9:87 - error TS2371: A parameter initializer is only allowed in a function or constructor implementation.

9 export function createIcons({ icons: IconsObj, nameAttr: string = 'icon-name', attrs: Attributes = {} }): VoidFunction;
                                                                                        ~~~~~~~~~~
```
![image](https://user-images.githubusercontent.com/21136680/112497405-35661400-8da3-11eb-8fce-b938ac97b243.png)

**Comment by ericfennis** [comment]
@SMAH1 Thank for posting this. I will look in to it.

**Comment by ericfennis** [comment]
@SMAH1 I'm working on this. Sorry I didn't test it properly. Hope to rollout the fix to today.

**Comment by ericfennis** [comment]
Fixed in [v0.15.1](https://github.com/lucide-icons/lucide/releases/tag/v0.15.1)
